### ISSUE TITLE
kernel specific modules not reflected in start file

### ISSUE THREAD

**Issue by elwer** [NormanTUD/JupyterKernels#2]
Kernel specific modules in the form:
```
tensorflow": {
      "name": "TensorFlow (Machine Learning)",
      "modules_load": {
              "alpha": "TensorFlow/2.11.0-CUDA-11.7.0",
              "romeo": "TensorFlow/2.11.0-CUDA-11.7.0",
              "barnard": "TensorFlow/2.9.1"
      },
```

are not reflected in the start-kernel.sh file

**Comment by elwer** [comment]
Just have seen: there is one possible parameter called "module_load" (e.g. `"module_load": ["PyTorch/1.13.1"]`), while the above one is "modules_load". Is that intended or should they be unified?

**Comment by NormanTUD** [comment]
The first issue should be solved, the 2nd one is intentional. modules_load allows you to load different modules, depending on the cluster host. The module_load does not allow that, it uses the modules listed there indiscriminately of the cluster.

Hopefully that solves this.

**Comment by elwer** [comment]
Now, in case we don't have any particular modules defined via `module_load` or the new `module_load_cluster` parameter, it adds "null" to the list of modules. Can we have a check in advance whether the parameters are defined and if not, skip it? "null" should be avoided.

**Comment by NormanTUD** [comment]
Please see if its fixed in e1097b399adefad73a98055d5c9291c3b6063099
